lib/package.json

```diff
@@ -27,7 +27,7 @@
   },
   "dependencies": {
     "@storyblok/js": "^2.2.2",
-    "svelte": "^3.59.2"
+    "svelte": "^4.0.0"
```

Comment:
Could we already update to the _most_ recent version (4.0.5) or is that unnecessary?